src/modules/ie-objects/helpers/isStringOfFormatIeObjectSchemaIdentifier.ts

```diff
@@ -1,3 +1,3 @@
 export function isStringOfFormatIeObjectSchemaIdentifier(id: string): boolean {
-	return /^[0-9a-f]{64}$/g.test(id);
+	return /^[0-9a-f]{64}[0-9a-f]*$/g.test(id);

```

Comment:
This is the same as `/^[0-9a-f]{64,}$/g`, https://regex101.com/r/j39nU6/1

Reply:
i swear i tried that one first, but it didn't work
now i try it and it does work
mush have done something wrong
thx